examples/ec2/subnet.yaml

```diff
@@ -6,18 +6,18 @@ spec:
   forProvider:
     region: us-west-1
     availabilityZone: us-west-1b
-    vpcIdRef:
+    vpcidRef:
       name: sample-vpc
     cidrBlock: 172.16.10.0/24
 ---
-apiVersion: ec2.aws.jet.crossplane.io/v1alpha1
+apiVersion: ec2.aws.jet.crossplane.io/v1alpha2
 kind: Subnet
 metadata:
   name: sample-subnet2
 spec:
   forProvider:
     region: us-west-1
     availabilityZone: us-west-1c
-    vpcIdRef:
+    vpcidRef:
```

Comment:
Why don't we have `vpcIdRef` as the ref name here? We may need to check the `subnet` resource to see why it does not conform to our conventions. Could you please open an issue to investigate?

Reply:
I checked EC2 and Route53 subresources and all of them are refering VPCs as `vpcidRef`. Opened this [issue](https://github.com/crossplane-contrib/provider-jet-aws/issues/153) for further discussion.